### PR TITLE
Update to point to Syracuse.io

### PR DESCRIPTION
**Pull request by vormwald**
Since the OpenHack Syracuse slack group has been moved over to syracuse.io, lets update the message on the "chat local" page.

### CODE REVIEW

File: slack.md [hunk 1]
```diff
@@ -5,31 +5,10 @@ title: Chat Local
 header-img: 'img/slack-action.jpg'
 ---
 
-Come hang out with local Syracuse developers in the OpenHack Syracuse Slack
-group.
+Come hang out with local Syracuse developers in the OpenHack Syracuse Slack.
```
Comment (myrridin) [inline slack.md:8]: Might want to specifiy channel here. Reads a little weird. i.e.

`Come hang out with local Syracuse developers in the OpenHack Syracuse Slack channel.`

File: slack.md [hunk 2]
```diff
@@ -5,31 +5,10 @@ title: Chat Local
...
-  <div id="success"></div>
-  <div class="row">
-    <div class="form-group col-xs-12">
-      <button type="submit" class="btn btn-default">Send</button>
-    </div>
-  </div>
-</form>
-
-*No spam, we promise, cancel anytime!*
+Also, here's a dog gif, lol ![](http://i.giphy.com/yXBqba0Zx8S4.gif)
```
Comment (myrridin) [inline slack.md]: This seems a bit weird, but I like the humor. Only suggestion would be to remove `lol`.

File: slack.md [hunk 3]
```diff
@@ -5,31 +5,10 @@ title: Chat Local
 header-img: 'img/slack-action.jpg'
 ---
 
-Come hang out with local Syracuse developers in the OpenHack Syracuse Slack
-group.
+Come hang out with local Syracuse developers in the OpenHack Syracuse Slack.
 
-This is a place to discuss the local development scene, seek help, share advice,
-and discuss everything related to being a Software / Web person in Syracuse NY.
+You can find us in the `#openhack-meetup` group on the local
```
Comment (vormwald) [inline slack.md:10]: Here is a comment.... lol.

**Comment by vormwald** [comment]
Yah I should have mentioned that this was done as an example PR for the StartFast Code students.  Thus, the dog gif :)

**Pull request merged**